<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YoukaiTabForm</class>
 <widget class="QWidget" name="YoukaiTabForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>基本</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>番号1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="num1SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>番号2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="num2SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>不明1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>不明2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="unk1SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="unk2SB">
            <property name="maximum">
             <number>65535</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>妖怪の種類</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="youkaiCB">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>ニックネーム</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="nicknameEdit">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>[ルビを振る文字/ルビ]</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>レベル</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="levelSB">
            <property name="minimum">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="IDEdit">
            <property name="inputMask">
             <string notr="true">HHHHHHHH</string>
            </property>
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>現在レベルでの経験値</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="expSB">
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string> 個体値</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>HP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="IV_HPSB">
            <property name="maximum">
             <number>80</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>ちから</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="IV_StrSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>ようりょく</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="IV_SprSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>まもり</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="IV_DefSB">
            <property name="maximum">
             <number>40</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>わざ</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="move1CB"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="move2CB"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="move3CB"/>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="defaultAtkB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>デフォルト</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>フラグ</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="flag0CB">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag1CB">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag2CB">
              <property name="text">
               <string>New</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag3CB">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QCheckBox" name="flag4CB">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag5CB">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag6CB">
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="flag7CB">
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="youkaiNumApplyB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>妖怪番号更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateIDButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ID更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fixIVButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>個体値修正</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
